src/components/Spinner/README.md

```diff
@@ -0,0 +1,12 @@
+# Spinner
+
+[View story source](https://github.com/balena-io-modules/rendition/blob/master/src/components/Spinner/story.js)
+
+## Props
+
+| Name          | Type      | Default   | Required   | Description                                          |
+| ------ | ------ | --------- | ---------- | ------------- |
+| `children`     | `React.ReactNode` |         | -          | If the spinner has children, it will show an opaque overlay over the children when it is spinning |
+| `spinning`     | `boolean`         | true    | -          | If passed, it will control whether the spinner is shown or not |
```

Comment:
this can be renamed as hidden. `spinning: false` sounds like it is rendered but it won't be spinning.

Reply:
...or `active`? I'm not too familiar with Rendition standards but 'positive' boolean properties are usually a bit more intuitive than 'negative' boolean ones (with the notable exception of 'disabled'). 

To be honest I read the description and assumed `spinning` meant that, if set, the spinner would be spinning (rotating) _in addition to_ the label and any other spinner UI.

Reply:
@iamsolankiamit @grahammcculloch I just borrowed the naming from a ant design, but I agree that it can be made better. I think `show` is the most descriptive in this case, what you think?